### DISCUSSION TITLE
Reconstruction of 3D fault plane: iso-resistivity curves from slices of  2D inverted ERT images?

### DISCUSSION THREAD

**Discussion by Chih-Hao-Kuo** [gimli-org/pyGIMLi#694] [General]
I have 2D inverted resistivity images at 10 closely spaced locations. I have latitudes and longitudes for these 10 locations. Is it possible to interpolate iso-resistivity curves in order to estimate geological fault location/orientation?
![東部縱谷瑞穗斷層系統構造 Ruisui Fault](https://github.com/gimli-org/gimli/assets/66291110/7ba72b48-8bfd-4251-8f2e-82e60ef280d1)

**Comment by halbmy** [comment]
No idea how exactly the data base is. I would suggest a geostatistical approach. One could probably use an approach like in Martin et al. (2020). 

Martin, T., Kuhn, K., Günther, T. & Kniess, R. (2020): Geophysical exploration of a historical stamp mill dump for volume estimation of valuable residues. J. Env. Eng. Geophys. 25(2), 275-286, doi:10.2113/JEEG19-080.
